### DISCUSSION TITLE
how to delete the files after unregistering a harvest.

### DISCUSSION THREAD

**Discussion by tridoxx** [GetDKAN/dkan#3904] [Q&A]
Hello, I have unregistered a harvest and I see that if I go to resources, the files are kept, in the distribution folder, is there any way to delete them? thank you

**Comment by janette** [comment]
If you go to /admin/dkan/resources
there is a checkbox for "Delete local resource", if you check this then the copies of remote files will be removed from your web server.

Before you deregister a harvest, you can run this command to delete the datasets in the harvest. Follow it up with cron to clean up the distributions.
`drush dkan:harvest:revert {harvest_id}`

**Comment by tridoxx** [comment]
Hello, thank you very much, in the first part you talk about deleting the copy of the remote file, but if I have a harvest plan and I delete that copy of the remote file, it does not generate any problem, which could be when downloading the files? or the files that are stored when downloading the resource with the "download" button from the front are saved elsewhere?, thank you very much
